lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_state.cpp

```diff
@@ -59,12 +59,16 @@ void put_sgw_ue_state(sgw_state_t* sgw_state, imsi64_t imsi64) { return; }
 
 void delete_sgw_ue_state(imsi64_t imsi64) { return; }
 
-void sgw_free_s11_bearer_context_information(
-    sgw_eps_bearer_context_information_t** sgw_eps_context) {
-  if (*sgw_eps_context) {
-    sgw_free_pdn_connection(&(*sgw_eps_context)->pdn_connection);
-    delete_pending_procedures((*sgw_eps_context));
+void sgw_free_s11_bearer_context_information(void** ptr) {
+  if (!ptr) {
+    return;
+  }
+  sgw_eps_bearer_context_information_t* sgw_eps_context =
+      reinterpret_cast<sgw_eps_bearer_context_information_t*>(*ptr);
+  if (sgw_eps_context) {
+    sgw_free_pdn_connection(&sgw_eps_context->pdn_connection);
+    delete_pending_procedures(sgw_eps_context);
   }
-  free_wrapper((void**)sgw_eps_context);
+  free_wrapper(reinterpret_cast<void**>(ptr));
```

Comment:
```suggestion
  free_cpp_wrapper(reinterpret_cast<void**>(ptr));
```

Reply:
In this PR, memory for sgw_eps_bearer_context_information_t is allocated with calloc. But in other PR, memory is allocated with new(). It's here, https://github.com/magma/magma/pull/13895/files#diff-89dd268674e287dc1dc45f2515f69e41f864be6074f3e03529324cb1eeecb374R189